src/box/iproto.cc

```diff
@@ -139,6 +140,10 @@ struct iproto_thread {
 	 * List of stopped connections
 	 */
 	RLIST_HEAD(stopped_connections);
+	/*
+	 * List of all connections
+	 */
+	RLIST_HEAD(all_connections);
```

Comment:
here is better `struct rlist all_connections`, initialization in declaration is valid in c++11, but better `struct rlist all_connections`. I fexed previous string in separate patch.

Reply:
Fix